tools/mo/requirements_mxnet.txt

```diff
@@ -1,9 +1,4 @@
-mxnet~=1.2.0; sys_platform == 'win32'
-mxnet>=1.7.0.post2,<=1.9.1; sys_platform != 'win32'
-networkx~=2.5; python_version <= "3.6"
-networkx<=2.8.8; python_version > "3.6"
-numpy>=1.16.6,<1.25.0
-defusedxml>=0.7.1
-urllib3>=1.26.4
-requests>=2.25.1
-fastjsonschema~=2.15.1
+-c ../../src/bindings/python/constraints.txt
+-r ./requirements.txt
+mxnet
+urllib3
```

Comment:
mxnet, urllib3 should have versions here

Reply:
`Add constraints.txt which allows to keep dependency versions in a single place` - FYI